script.js

```diff
@@ -29,43 +28,41 @@ items[2].element.alt = "item3";
 items[3].element.src = "type4.svg";
 items[3].element.alt = "item4";
 items.push({element: document.createElement('img'), x: 1000, y: 80, onBelt:true, type: 'type5'});
+items[4].element.id = '5';
```

Comment:
It seems like you're now using the index of the item as the item's "ID" for putting them in bins - if that's the case, it might be worth changing the values of `.id` here in this function to make sure they're the same as the index. Right now you've got 1,2...6, but maybe those could get changed to 0,1,2...5? Your items need _some_ kind of unique identifier, and I think using the index as that id is fine, but being consistent will probably help you avoid some nasty bugs in the future.